BuildResidentialHPXML/measure.rb

```diff
@@ -1145,6 +1145,12 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('Frac')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_crankcase_watts', false)
+    arg.setDisplayName('Cooling System: Crankcase Power Watts')
+    arg.setDescription("Cooling system crankcase power consumption in Watts. Applies only to #{HPXML::HVACTypeCentralAirConditioner}, #{HPXML::HVACTypeMiniSplitAirConditioner}, #{HPXML::HVACTypePTAC} and #{HPXML::HVACTypeRoomAirConditioner}. If not provided, the OS-HPXML default is used.")
+    arg.setUnits('W')
```

Comment:
I hate to request this, but I think the terminology should be improved. "Crankcase power" is kind of confusing, it's really "crankcase heater power". Could you update this entire PR (HPXML elements, variable names, docs, etc.) to better reflect that this input refers to a crankcase _heater_?

Reply:
No problem! It's totally fine to request changes like this. :)